packages/react/src/components/Forms/Radio/index.scss

```diff
@@ -5,6 +5,7 @@
     width: var(--c--components--forms-checkbox--size);
     height: var(--c--components--forms-checkbox--size);
     border: 1.5px solid var(--c--components--forms-radio--border-color);
+    background-color: white;
```

Comment:
By seeing this modification isolated, I think it could be great to use a token to set this color (to do also on checkbox component). (`--c--components--forms-checkbox--background-color`)